### PR TITLE
Added Customized ETCD v3.4.4.1 into arktos repo

### PR DESCRIPTION
**Pull request by jshaofuturewei**
As title

### CODE REVIEW

**Comment by Sindica** [comment]
Do we want to directly replace the 3.4.4 community binary? How do we differentiate this is a community version or customized version?

**Comment by Sindica** [comment]
I would suggest adding customized ETCD as another folder and make soft link to it at arktos-up

**Comment by Sindica** [comment]
I just tested, you can use 3.4.4.1 as etcd version. Let's check with Xiaoning to see whether it is ok

**Comment by jshaofuturewei** [comment]
> Do we want to directly replace the 3.4.4 community binary? How do we differentiate this is a community version or customized version?

For community binaries, we can download them in the directory of third_party as it used to be.
For customized ones, I created a new folder build for them.

**Comment by jshaofuturewei** [comment]
> I would suggest adding customized ETCD as another folder and make soft link to it at arktos-up

Do you mean a folder out of third_party? 
The soft link has already created. It is third-party/etcd -> third-party/build/etcd_v3.4.4_linux_amd64.
It used to be third-party/etcd -> third-party/etcd_v3.4.4_linux_amd64(downloaded binaries)

**Comment by Sindica** [comment]
> > I would suggest adding customized ETCD as another folder and make soft link to it at arktos-up
> 
> Do you mean a folder out of third_party?
> The soft link has already created. It is third-party/etcd -> third-party/build/etcd_v3.4.4_linux_amd64.
> It used to be third-party/etcd -> third-party/etcd_v3.4.4_linux_amd64(downloaded binaries)

Yes. from the link or version, we cannot tell which version we are using. There needs to be something that allows us to tell which version we are using.

File: hack/lib/etcd.sh [hunk 1]
```diff
@@ -131,11 +131,11 @@ kube::etcd::install() {
       ln -fns "etcd-v${ETCD_VERSION}-darwin-amd64" etcd
       rm "${download_file}"
     else
-      url="https://github.com/coreos/etcd/releases/download/v${ETCD_VERSION}/etcd-v${ETCD_VERSION}-linux-${arch}.tar.gz"
-      download_file="etcd-v${ETCD_VERSION}-linux-${arch}.tar.gz"
+      url="https://github.com/futurewei-cloud/etcd/releases/download/v${ETCD_VERSION}/etcd-v${ETCD_VERSION}-linux-arm64.tar.gz"
```
Comment (Sindica) [inline hack/lib/etcd.sh:134]: amd64? should you check os value?

**Comment by jshaofuturewei** [comment]
> > > I would suggest adding customized ETCD as another folder and make soft link to it at arktos-up
> > 
> > 
> > Do you mean a folder out of third_party?
> > The soft link has already created. It is third-party/etcd -> third-party/build/etcd_v3.4.4_linux_amd64.
> > It used to be third-party/etcd -> third-party/etcd_v3.4.4_linux_amd64(downloaded binaries)
> 
> Yes. from the link or version, we cannot tell which version we are using. There needs to be something that allows us to tell which version we are using.

EXPORT ETCD_VERSION=3.4.4.1？We can see version in the third_party directory.

**Pull request merged**